README.md

```diff
@@ -45,30 +45,21 @@ jobs:
         id: meta
         with:
           images: YOUR_IMAGE
+
       - name: Cache var-cache-apt
         id: cache-var-cache-apt
         uses: actions/cache@v3
         with:
           path: var-cache-apt
           key: var-cache-apt-${{ hashFiles('Dockerfile') }}
-      - name: Cache var-lib-apt
```

Comment:
var-lib-apt is not a duplicate of var-cache-apt

Reply:
Oh, okay. The code is so terse, it is hard to read. Is this boilerplate repetition needed because of not supporting a array API? Why not having an array of source to target mapes?

Reply:
idk, maybe ask in https://github.com/actions/cache/discussions ?

Reply:
actions/cache supports multiple paths in one call. It seems buildkit-cache-dance lacks this support

Reply:
Contribution is wanted to add the support